src/web/mjs/connectors/ComicEarthStar.mjs

```diff
@@ -1,71 +1,94 @@
-import Publus from './templates/Publus.mjs';
-import Manga from '../engine/Manga.mjs';
+import CoreView from './templates/CoreView.mjs';
 
-export default class ComicEarthStar extends Publus {
+export default class ComicEarthStar extends CoreView {
 
     constructor() {
         super();
         super.id = 'comicearthstar';
         super.label = 'コミック アース・スター (Comic Earth Star)';
         this.tags = [ 'manga', 'japanese' ];
-        this.url = 'https://www.comic-earthstar.jp';
+        this.apiURL = 'https://comic-earthstar.com/graphql';
+        this.url = 'https://comic-earthstar.com';
     }
 
-    async _getMangaFromURI(uri) {
-        let id = uri.pathname.match(/\/([^/]+)\/?$/)[1];
-        let request = new Request(this.url + '/json/contents/detail/' + id + '.json', this.requestOptions);
-        let data = await this.fetchJSON(request);
-        let title = data.categorys.comic_category_title;
-        return new Manga(this, id, title);
-    }
+    async _getMangas() {
+        const mangalist = [];
 
-    /**
-     *
-     */
-    _getMangaList( callback ) {
-        let request = new Request( this.url + '/json/contents/top/default.json', this.requestOptions );
-        this.fetchJSON( request )
-            .then( data => {
-                let ongoingList = data.serial_comics.map( comic => {
-                    return {
-                        id: comic.comic_category_code,
-                        title: comic.title
-                    };
-                } );
-                let completedList = data.comple_comics.map( comic => {
-                    return {
-                        id: comic.comic_category_code,
-                        title: comic.comic_category_code.toUpperCase()
-                    };
-                } );
-                let mangaList = ongoingList.concat( completedList );
-                callback( null, mangaList );
-            } )
-            .catch( error => {
-                console.error( error, this );
-                callback( error, undefined );
-            } );
-    }
+        let operationName = 'Earthstar_Oneshot';
```

Comment:
I wouldn't use `let` because this variable is not stricly related to the next stage
I'd use an object as i believe it serves as self documentation (but perhaps that's personal taste)
```
const oneShotsRequest = {
    operationName = '',
    query= ''
}
```

Reply:
Graphql is pretty much straithforwarded tho :
A query string
A strign for operation name
An object as variables

Separating them may prevent dev errors when it comes to serialization and integrity after..